cpp/src/parquet/arrow/schema_internal.cc

```diff
@@ -181,7 +181,8 @@ Result<std::shared_ptr<ArrowType>> FromInt64(const LogicalType& logical_type) {
 
 Result<std::shared_ptr<ArrowType>> GetArrowType(Type::type physical_type,
                                                 const LogicalType& logical_type,
-                                                int type_length) {
+                                                int type_length,
+                                                const ::arrow::TimeUnit::type& int96_arrow_time_unit) {
```

Comment:
Same comment here, with respect to passing by value vs. reference.

Reply:
Done.